src/lib/walletAgendaPushSync.ts

```
@@ -0,0 +1,62 @@
+  return {
+    aps: { "content-available": 1 },
+    passSerialNumber: serialNumber,
+  };
+}
+
+/**
+ * Triggers silent APNs push notification to Apple Wallet pass upon itinerary modification (#4671).
+ */
+export function syncItineraryToWalletPass(
+  request: WalletPushSyncRequest
+): WalletPushSyncResult {
```

ESLint

[error] Replace `⏎··request:·WalletPushSyncRequest⏎` with `request:·WalletPushSyncRequest`

(prettier/prettier)